openmeter/subscription/connector.go

```diff
@@ -0,0 +1,388 @@
+package subscription
```

Comment:
apply please adapter-service-entity pattern similar to:

https://github.com/openmeterio/openmeter/tree/main/openmeter/app
https://github.com/openmeterio/openmeter/tree/main/openmeter/billing
https://github.com/openmeterio/openmeter/tree/main/openmeter/customer